docs/release_notes/0.101.0.md

```diff
@@ -0,0 +1,16 @@
+# Release 0.101.0
+
+## Features
+
+- Add tolerations for taints on NVIDIA specific node groups (#5345)
+- Add support for managed bottlerocket GPU nodes (#5059)
+- Parallel drain all nodegroups (#5146)
+
+## Improvements
+
+- Do not require overrideBootstrapCommand when ami family is Windows (#5248)
+- Backfill create cluster unittests (#5314)
```

Comment:
This was for improving our test coverage and does not have to go out in the release notes.